projects/storefrontapp/server.ts

```diff
@@ -15,14 +16,15 @@ const express = require('express');
 
 const ssrOptions: SsrOptimizationOptions = {
   concurrency: 20,
-  timeout: Number(process.env.SSR_TIMEOUT ?? 3000),
+  timeout: 3000,
+  reuseCurrentRendering: true,
```

Comment:
Do we plan to revert those 2 lines?

Reply:
Good catch, I'll revert just timeout. But I think it's for the best if we use this new option ourselves.